modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18557,6 +18557,7 @@ this-document-type-does-not-belong-to-this-site.-you-may-affect-other-sites-if-y
 this-editor-is-not-optimized-for-mobile-or-400-zooming=This editor is not optimized for mobile or 400% zooming.
 this-event-and-any-future-events-in-the-series-will-be-modified=This event, and any future events in the series, will be modified.
 this-event-does-not-repeat=This event does not repeat.
+this-external-reference-code-is-already-in-use=This External Reference Code is already in use.
```

Comment:
Please, change `This External Reference Code is already in use.` to `This external reference code is already in use.`

Reply:
Done!